tensorflow/tensorflow.bzl

```diff
@@ -1923,7 +1925,7 @@ def tf_py_wrap_cc(
         linkopts = extra_linkopts,
         linkstatic = 1,
         deps = deps + extra_deps,
-        data = tf_binary_additional_srcs() + tf_binary_additional_srcs(fullversion = True),
+        data = tf_binary_additional_srcs() + tf_binary_additional_srcs(fullversion = True) + tf_binary_additional_srcs(unversioned = True),
```

Comment:
why dont we edit tf_binary_additional_srcs so return all?
I really don't think calling this function 4 times is the way to go. We need to call this once, and have it return all needed files.
This seems to only fix the python packaging, but we just detected problems across the board. Our libtensorflow packages are also not including the "libtensorflow_framework.so" symlink, and I think this only fixes py_wrap_cc rules?

Reply:
Cant just return all always, only the full versioned one is an actual cc_library, the others are genrules and bazel gets upset depending on where its called and added to.

I originally wanted to have the .so dep on the .so.1 which would dep on the .so.1.13.1. but again cuz they arent cc_libraries, bazel doesnt like it. 

Do you have a better idea for what to name the args? having an arg to make it return all would be okay for places like this it just has to be an arg cuz it wont work everywhere.

alternatively, since the lib is a cc_binary (as opposed to cc_library), it does have a data= arg for runtime deps that we could add the .so and .so.1 symlinks to. I seem to recall having tried that but I cant remember if or what the issue with that was.

Reply:
What if we wrote a counterpart to this. `tf_binary_additional_data_deps()`.
Any rule that has a call to `tf_binary_additional_srcs` also has a call to `tf_binary_additional_data_deps` to add to its "data" dependencies. The data deps returns the symlinks, and the srcs returns all the actual shared objects. Bazel should be ok with that.